### PR TITLE
Small changes in readme and plotting of palettes

### PR DESCRIPTION
**Pull request by jromanowska**
Just some small things I saw that might need change. Will test more.

### CODE REVIEW

File: R/palette.R [hunk 1]
```diff
@@ -82,8 +82,9 @@ plot.palettehex <- function(x, ...){
   )
   for(i in 1:length(colr)){
     tmp <- colr[[i]]
-    graphics::text(1, i+.1, tolower(tmp$name[[1]]), cex = .7, pos = 2)
-    graphics::text(1, i-.1, tmp$id[[1]], cex = .7, pos = 2)
+    graphics::text(1, i+.1,
+                   paste0(tolower(tmp$name[[1]]), " (", tmp$id[[1]], ")"),
```
Comment (drmowinckels) [inline R/palette.R]: this is more of a code preference for me, but can you change the `paste0` to `sprintf`? To me it reads clearer what the end string should look like then

`sprintf("%s (%s)", tolower(tmp$name[[1]]), tmp$id[[1]])`

**Comment by drmowinckels** [comment]
This is lovely, thank you for taking a look. Please also add yourself as a contributor in the DESCRIPTION file :)

File: R/palette.R (hunk 1 above)
Comment (jromanowska) [reply R/palette.R]: Sure! :)

**Comment by drmowinckels** [comment]
thank you!

**Pull request merged**